rules/macos/credential_access_potential_ssh_bruteforce.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/16"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of macOS SSH KeyGen process execution from the same host. This may indicate an SSH brute force
+attempt to gain access to accounts when passwords are unknown.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential SSH Bruteforce Detected"
+references = ["https://themittenmac.com/detecting-ssh-activity-via-process-monitoring/"]
+risk_score = 47
+rule_id = "ace1e989-a541-44df-93a8-a8b0591b63c0"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Credential Access"]
+type = "threshold"
+
+query = '''
+event.category:process and event.type:(start or process_started) and process.name:"sshd-keygen-wrapper" and process.parent.name:launchd
```

I don't think `event.type:process_started` is needed anymore. It's just`event.type:start`.
```suggestion
event.category:process and event.type:start and process.name:"sshd-keygen-wrapper" and process.parent.name:launchd
```